code/modules/jobs/job_types/paramedic.dm

```diff
@@ -11,8 +11,8 @@
 
 	outfit = /datum/outfit/job/paramedic
 
-	access = list(ACCESS_MEDICAL, ACCESS_MORGUE, ACCESS_SURGERY, ACCESS_CHEMISTRY, ACCESS_GENETICS, ACCESS_CLONING, ACCESS_VIROLOGY, ACCESS_MINERAL_STOREROOM, ACCESS_MAINT_TUNNELS)
```

Comment:
is it necessary to remove these for low-pop access?

Reply:
Most station maps have a medical apothecary room anyway.
And as for virology, it's really not pertinent to the job.